roles/edpm_nova_compute/tasks/configure.yml

```diff
@@ -1,131 +0,0 @@
----
-# Copyright 2022 Red Hat, Inc.
-# All Rights Reserved.
-#
-# Licensed under the Apache License, Version 2.0 (the "License"); you may
-# not use this file except in compliance with the License. You may obtain
-# a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
-# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
-# License for the specific language governing permissions and limitations
-# under the License.
-
-- name: Manage PMEM namespaces for vPMEM
-  ansible.builtin.include_role:
-    name: edpm_nvdimm
-  when: edpm_nova_compute_pmem_namespaces  # not a bool
-
-- name: Manage OS version 9 specific sysctls
-  when:
-    - ansible_facts['os_family'] == 'RedHat'
-    - ansible_facts['distribution_major_version'] is version('9', '==')
-  block:
-    - name: Enable post-copy by setting unprivileged_userfaultfd
-      ansible.posix.sysctl:
-        name: "vm.unprivileged_userfaultfd"
-        value: 1
-        sysctl_set: true
-        sysctl_file: /etc/sysctl.d/99-edpm-postcopy.conf
-        state: present
-        reload: true
-
-- name: Enable/disable ksm
```

Comment:
these are not currently used and will be readded as new roles that will be run by the dataplen operator.

ksm has nothing to do with nova or libvirt so its inclusion in this role was incorrect.

i or others form the compute dfg will read this and enable it via the dataplane operator in Q2 assuming we plan to continue to support this going forward.

Reply:
or put another way i intentionaly did not include this in the playbook in nova when i was writing it  because i did not think it should belong in the nova role in the first place.

the intent is to introduce a generic sysctl role and a ksm role.

those will handle ksm and turning sysctl.

Reply:
> > I'd also want to see an ACK from neutron/ovn folks that they plan to talk on the configuration work instead of just claiming so in the commit message.
> 
> we exlcivily use the nova user to talk to all service including placement so the neutron user which is create the same way in the same project with the same permission can also talk to nova and placment.

I can't understand what you've written, so let's hold off on moving forward with this PR until there is common understanding.

Reply:
we'll hold off on deleting tasks that are still needed until there is a replacement.